src/buildx/install.ts

```diff
@@ -33,13 +35,18 @@ import {GitHubRelease} from '../types/github';
 
 export interface InstallOpts {
   standalone?: boolean;
+  gitAuthToken?: string;
 }
 
 export class Install {
   private readonly _standalone: boolean | undefined;
+  private readonly _gitAuthToken: string | undefined;
+  private readonly _octokit: InstanceType<typeof Octokit>;
 
   constructor(opts?: InstallOpts) {
     this._standalone = opts?.standalone;
+    this._gitAuthToken = opts?.gitAuthToken;
+    this._octokit = github.getOctokit(this._gitAuthToken || '');
```

Comment:
Let's avoid creating a new octokit client and reuse one from GitHub class: https://github.com/docker/actions-toolkit/blob/ca519e1aa876dc868f4d313c6c9cf25b3edc0be1/src/github.ts#L29